src/richie/apps/courses/templatetags/extra_tags.py

```diff
@@ -216,4 +216,7 @@ def has_connected_lms(course_run):
     This enables our templates to either use the <CourseRunEnrollment /> component or a simple
     link to the course run.
     """
-    return LMSHandler.select_lms(course_run.resource_link) is not None
+    if course_run.resource_link:
+        return LMSHandler.select_lms(course_run.resource_link) is not None
```

Comment:
I think it's better to fix the error at its root ie in the `select_lms` method so that all other callers can benefit from the fix.

Reply:
Yep of course ! 👍